src/parser_urdf.cc

```diff
@@ -1343,6 +1343,7 @@ void URDF2SDF::ParseSDFExtension(TiXmlDocument &_urdfXml)
       else if (childElem->ValueStr() == "turnGravityOff")
```

Comment:
To avoid introducing a new variable `isGravity`, another option could be adding another `if` statement after this one. For example, 

```C++
else if (childElem->ValueStr() == "gravity")
{
  std::string valueStr = GetKeyValueAsString(childElem);

  if (lowerStr(valueStr) == "false" || lowerStr(valueStr) == "no" ||
      valueStr == "0")
  {
    sdf->gravity = false;
  }
  else
  {
    sdf->gravity = true;
  }
}
```

Reply:
That is true; if we wanted to avoid adding a new variable, we could add some extra code instead. I personally lean towards less code, but I'm open to change this if there is consensus for that

Reply:
Less code sounds good to me :)

Reply:
Less code sounds good to me :)